src/components/Inputs/TextInput/TextInput.js

```diff
@@ -108,12 +111,18 @@ PrivateTextInput.PUBLIC_PROPS = {
   validator: PropTypes.func,
   onBlur: PropTypes.func,
   onFocus: PropTypes.func,
+  restrictIllegal: PropTypes.bool,
```

Comment:
We need to be able to allow special characters for password. Reality is we urlencode at the KOA side anyway so this should be fine I think.